### DISCUSSION TITLE
Account Suspended And not able to access it also not getting any reply from support

### DISCUSSION THREAD

**Discussion by Vaishnavipatel26** [community/community#134293] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Account Name: UDAIBHAT
link to profile : https://github.com/UDAIBHAT

My Account has been suspended and i am unable to contact support through that acoount due access denial , so i am contacting you through this secodary account , Please help me out as i am unable to access my account

i have also recieved the password reset link from github stating that my account has undergone a suspecious login activity resulting in freezing my account , but through the link provided i am unable to reset my password as it is showing "Access to your account has been suspended due to voilation of terms and condition "
   

I have also created the two Tickets in github support but its been a week and got no reply from github 
Ticket 1 : #2896220
Ticket 2 : #2898013

Plz help me out my work is on github as i am a student with placements on ogoing i am unable to show my projects Please help me out 

Neither passwork reset is working nor recieving any reply from support PLEASE HELP please

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Vaishnavipatel26, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
